integration/config/core.yaml.orig

```diff
@@ -1,746 +0,0 @@
-# Copyright IBM Corp. All Rights Reserved.
-# Copyright Intel Corp. All Rights Reserved.
```

Comment:
file not used?

Reply:
this is not used and got in only by accident in an earlier PR